src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -33,9 +33,23 @@ public interface IPreferences
         ConnectorType ConnectorType { get; set; }
 
         /// <summary>
-        /// Indicates whether background preview is active or not.
+        /// Collection of pairs [BackgroundPreviewName;isActive]
         /// </summary>
-        bool IsBackgroundPreviewActive { get; set; }
```

Comment:
@QilongTang @ramramps (sending you guys my notes - so I'm not just talking to myself ) 

I'm a bit worried about this since we are now changing the interface after our 1.0 release. The api for others (not sure if anyone) are using the preferences interface will have trouble.

Other worry is loading currently saved preferences.